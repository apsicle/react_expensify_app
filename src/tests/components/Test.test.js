import React from 'react';
import { shallow } from 'enzyme';

class TestComponent extends React.Component {
    constructor(props) {
        super(props);
        
    this.state = {
        description: 'asdf123'
      };
    }

    render() {
        return (
            <div>
                <p>{this.state.description}</p>
            </div>
        )
    }
}

test('asdf', () => {
    const wrapper = shallow(<TestComponent />);
    // console.log(wrapper.state().description); // cannot get property 'description' of undefined
    // console.log(wrapper.state()); // returns undefined
    expect(wrapper).toMatchSnapshot(); // I get <p>asdf123</p> in here, so state seems to be working?
});