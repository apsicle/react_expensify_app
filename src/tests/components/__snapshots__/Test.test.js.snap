// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`asdf 1`] = `
<div>
  <p>
    asdf123
  </p>
</div>
`;
